terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/token/transaction/HttpSessionTransactionTokenStore.java

```diff
@@ -277,9 +277,12 @@ public void store(TransactionToken token) {
     }
 
     /**
-     * [todo] write javadoc
-     * @param token
+     * Whether the {@link TransactionToken} with the same namespace and tokenkey in HTTP session.
```

Comment:
- [ ] ``Check whether the {@link TransactionToken} with the same namespace and token key exist in HTTP session.``